### ISSUE TITLE
Change cytoplasmic sides to cytosolic sides?

### ISSUE THREAD

**Issue by ukemi** [geneontology/go-ontology#17000]
I think it would be much more useful if we changed the terms and logical definitions of cytoplasmic sides of membranes to be cytosolic sides of membranes. We need to examine if this would cause any problems.

**Comment by deustp01** [comment]
Does it matter than some membranes don't have a cytosolic side, e.g., the specialized plasma membrane around a cilium is in direct contact with the ciliary structure - no intervening cytosol?

**Comment by ukemi** [comment]
@deustp01 I don't think this would be a problem because we would define it as a side of membrane and part-of some membrane and adjacent to some cytosol. We wouldn't create the class for the ciliary membrane. There are already a bunch of 'cytoplasmic' terms in the ontology. These are the ones we would need to examine. I have started a project for spatial relations and added this ticket to it.

**Comment by ukemi** [comment]
We do have the term 'cytoplasmic side of dendritic spine plasma membrane'. Would the cytoplasm of the dendritic spine be considered cytosol or does it suffer from @deustp01 's caveat above?

**Comment by BarbaraCzub** [comment]
I would probably say there is cytosol within dendritic spines.
Cilia are microtubule-based organelles, and the ciliary axoneme is a relatively thick cytoskeletal structure, wrapped by cell membrane quite tightly. The structure of a dendritic spine is based on much smaller actin filaments, so it is more variable, and there are more 'spaces' within it that could be cytosol-filled. The dendritic spine actually 'grows out' of a microtubule-based structure. So a cilium and a dendritic spine are not analogous. But going back to cilia, what about photoreceptor outer segments? Only the part of the axoneme forming the connecting cilium in the transition zone fits @deustp01 's caveat in this case, doesn't it?

**Comment by ukemi** [comment]
Thanks @BarbaraCzub. I think the cilium is a different case. I was specifically interested in the dendrite because of the current structure of the ontology.
